docker/data-science/notebooks/ch15/1.llm-exploration.ipynb

```diff
@@ -801,6 +807,118 @@
     "news_article_generated_labelled = get_generative_response(entities_prompt_news_article)\n",
     "extract_entities(news_article_generated_labelled)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Listing 15.14"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip install gdown\n",
+    "def open_image_file(movie_id):    \n",
+    "    url = 'https://drive.google.com/uc?id=0B9P1L--7Wd2vNm9zMTJWOGxobkU'\n",
```

Comment:
Need to remove Google Drive references. Should be loading embeddings internally instead.

Reply:
Stubbed code has been removed.